stock_picking_validate_jobcan_wf/models/stock_picking.py

```diff
@@ -36,7 +49,11 @@ def _compute_show_skip_jobcan_wf(self):
     def _compute_show_jobcan_wf_number(self):
         for pick in self:
             pick.show_jobcan_wf_number = False
-            if pick._is_outgoing() and not pick.skip_jobcan_wf:
+            if (
+                pick._is_outgoing()
+                and not pick.skip_jobcan_wf
+                and not pick._receipt_return_picking()
```

Comment:
Why do we need this adjustment?
```suggestion
```

Reply:
This adjustment is needed for not showing and requiring of `jobcan_wf_number` for receipts return.
https://github.com/qrtl/axls-custom/blob/3720a67608535eea4fdf2b787340386c8f26f041/stock_picking_validate_jobcan_wf/views/stock_picking_views.xml#L15-L18